api/jobs/job_handler.go

```diff
@@ -23,6 +27,35 @@ const workerImage = "radix-pipeline"
 // RadixJobTypeJob TODO: Move this into kube, or another central location
 const RadixJobTypeJob = "job"
 
+func StepNotFoundError(stepName string) error {
```

Comment:
Maybe `Step***Error` methods can be package private (start with not capital letter)?

Reply:
fixed